pkgs/development/interpreters/micropython/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, lib, fetchFromGitHub, pkgs }:
```

Comment:
all dependencies should be listed explicitly

Reply:
Fixed